challenges/release/02.md

```diff
@@ -40,20 +40,19 @@ If you want to store secret key in your repository like `SENTRY_API_KEY`, you ca
 To create a new secret, run :
 
 ```console
-eas secret
+eas env
 ```
 
 ```console
-eas secret:create  // create a new secret
-eas secret:list    // view any existing secrets for this project
+eas env:create  // create a new secret
+eas env:list    // view any existing secrets for this project
 ```
 
-- [ ] use `eas secret:create` and create a fake `SWAPI_KEY` because there is no API key for the Star Wars API.
+- [ ] use `eas env:create` and create a fake `SWAPI_KEY` because there is no API key for the Star Wars API.
 
 In the next lesson, we are going to build your app and show you how to install it on your phone.
 
 ## 👽 Bonus
 
-- [ ] Add a custom icon for your dev app
-
-Use [App-icon-badge](https://github.com/obytes/app-icon-badge) and update your `app.config.js` to have a custom icon for your dev app.
\ No newline at end of file
+- [ ] Check [with-env config](https://github.com/betomoedano/with-environments/blob/main/app.config.ts) and update your `app.config` file.
```

Comment:
you can add this

- [ ] use Figma to generate icons https://www.figma.com/community/file/1466490409418563617/expo-app-icon-splash-v2-community